content/guides/react/hooks/use-anchored-position.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Anchored position behavior
```

Comment:
It might be worth prominently mentioning that this is a private hook only used in `@primer/react`